src/components/templates/Embeddable/Embeddable.tsx

```diff
@@ -22,7 +22,7 @@ export const Embeddable: React.FC<EmbeddableProps> = ({ venue }) => {
         src={iframeUrl}
         className="embeddable__iframe"
         style={iframeStyles}
-        allow={IFRAME_ALLOW}
+        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture; fullscreen; camera; microphone;"
```

Comment:
Instead of hardcoding directly into the component like this, it would be better to extract a new `IFRAME_ALLOW_EMBEDDABLE` or similar in `src/settings` and use that here.

Reply:
Do we know that these settings are all required here? How were they chosen?

Reply:
I took the existing IFRAME_ALLOW settings and added the camera; microphone. I imagine that autoplay and fullscreen would be good to keep but cannot personally speak to why accelerometer, encrypted-media, gyroscope, or picture-in-picture would be needed

Reply:
@0xdevalias I updated per your requests